openshift/README.md

```diff
@@ -7,57 +7,132 @@ This project uses OpenShift container platform (v3.11) for hosting the applicati
-2 View the parameters `oc process --parameters pims-api-deploy`
-3 Create a deploy.env file that contains the values for the parameters within the template
-4 Create Objects `oc process pims-api-deploy --param-file=[.env] | oc create -f -`
+1. `oc create -f openshift/templates/app/pims-api/deploy.yaml`
+2. View the parameters `oc process --parameters pims-api-deploy`
+3. Create a deploy.env file that contains the values for the parameters within the template
+4. Create Objects `oc process pims-api-deploy --param-file=[.env] | oc create -f -`
 
 # Environments
 
 There are 4 environments setup in the OpenShift Platform.
 
-- Tools : Contains all the tools/external services required by the application.
-- Dev : Contains a running application instance of an open pull request branch.
-- Test : Contains a running application instance of the current release branch.
-- Prod : Contains a running application instance of the current state of master.
+- **Tools** - Contains all the tools/external services required by the application.
+- **Dev** - Contains a running application instance of an open pull request branch.
+- **Test** - Contains a running application instance of the current release branch.
+- **Prod** - Contains a running application instance of the current state of master.
 
 # Filename Convention
 
 Files are named with the following convention in mind:
 
-- [APPLICATION]-build.yaml: Build configuration for the application. Used for building any images required by the deployment configuration.
-
-- [APPLICATION]-deploy.json: Deployment configuration for the application. These deployment pods are long lived.
+| Object                    | Description                                                                                                     |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| [APPLICATION]-build.yaml  | Build configuration for the application. Used for building any images required by the deployment configuration. |
+| [APPLICATION]-deploy.yaml | Deployment configuration for the application. These deployment pods are long lived.                             |
 
 # CI/CD Tools
 
 ## Jenkins
 
-Jenkins is being used to orchestrate jobs between Github and OpenShift. Anytime a PR is open against the release/master branch it triggers a job on Jenkins which runs the pipeline stages and reports the status back to Github.
-
-[Link to Documentation](https://docs.openshift.com/container-platform/3.10/using_images/other_images/jenkins.html)
-
-[Link to the jenkins s2i image](https://github.com/BCDevOps/openshift-components/tree/master/cicd/jenkins)
+Jenkins is being used to orchestrate jobs between GitHub and OpenShift. Anytime a Pull Request is opened against the release/master branch, it triggers a job on Jenkins which runs the pipeline stages and reports the status back to GitHub.
```

Comment:
`release/master` probably should be `dev/master`

Reply:
fixed